src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -92,7 +92,8 @@ The .NET Foundation licenses this file to you under the MIT license.
       <NativeLibrary Include="$(IntermediateOutputPath)libs/System.Globalization.Native/build/libSystem.Globalization.Native.a" />
       <DirectPInvoke Include="libSystem.Globalization.Native" />
       <StaticICULibs Include="-Wl,-Bstatic" Condition="'$(StaticExecutable)' != 'true'" />
-      <StaticICULibs Include="-licuio -licutu -licui18n -licuuc -licudata -lstdc++" />
+      <StaticICULibs Include="-licuio -licutu -lstdc++" />
```

Comment:
Does/will HybridGlobalization work on Linux?

This is all under a `StaticICULinking=-true` condition that only works on Linux (ref: https://github.com/dotnet/runtime/issues/94951#issuecomment-1817640360).